### DISCUSSION TITLE
Is there a way to define a custom location for a theme?

### DISCUSSION THREAD

**Discussion by ylluminate** [wezterm/wezterm#6267] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I'd like to define a theme via an absolute (or better: a relative) path for WezTerm. Is there a way to configure the `config.color_scheme` to utilize `~/some/path/to/a/theme.tom`?

**Comment by akthe-at** [comment, marked as answer]
I believe you just require it (the module) and depending on where you have it saved compared to your config file you may need more or less of a path in the require statement (if your theme is called poo_brown.lua and its in the same dir as the config then its just `local brown_scheme = require "poo_brown"` at the top). From there you are just referring that local variable when setting `config.color_scheme`

**Reply by ylluminate** [reply]
Thanks, so to be explicit, this seems to work:
```lua
-- Pull in the wezterm API
local wezterm = require 'wezterm'

-- Load the color schemes from the TOML files:
local tokyonight = '/Users/user/path/to/themes/clones/tokyonight.nvim/extras/wezterm/tokyonight_storm.toml'

-- ... other config stuff ...

config.color_scheme = 'tokyonight_storm'

-- ... rest of config stuff ...
```

So I got the color_scheme name from the metadata section of this:
https://github.com/folke/tokyonight.nvim/blob/main/extras/wezterm/tokyonight_storm.toml
```toml
...

[metadata]
aliases = []
author = "folke"
name = "tokyonight_storm"
```
